services/avena_fm/scripts/avena_fm_demo_epy_block_1.py

```diff
@@ -0,0 +1,60 @@
+
+import os
+import numpy as np
+from gnuradio import gr
+import pmt
+from pynats2 import NATSClient
+import json
+
+class blk(gr.sync_block):  # other base classes are basic_block, decim_block, interp_block
+
+    def __init__(self, nats_server='nats://localhost:4222', 
+                 subject='sdr.control'):  # only default arguments here
+        gr.sync_block.__init__(
+            self,
+            name='NATS Subscription',   # will show up in GRC
+            in_sig=[np.complex64],
+            out_sig=None
+        )
+        # if an attribute with the same name as a parameter is found,
+        # a callback is registered (properties work, too).
+        self.port_1 = 'Frequency'
+        self.port_2 = 'Center Frequency'
+        self.port_3 = 'Gain'
+        self.port_4 = 'Stream'
+        self.subject = os.getenv('HOSTNAME') + '.' + subject
```

Comment:
AVENA_PREFIX?

Reply:
So, should we set AVENA_PREFIX to be equal to HOSTNAME by default?